.editorconfig

```diff
@@ -18,3 +18,7 @@ indent_size = 2
 
 [*.html]
 indent_size = 4
+
+[*.{tsx,ts,js,jsx,md}]
+indent_style = space
+indent_size = 2
```

Comment:
```suggestion
```
@kaf-lamed-beyt we are indenting with 4 spaces in all files except .yml files, this ensures consistency across files, `python` / `typescript`.

Reply:
Awesome! I committed the suggestion now.